### PR TITLE
docs(dpp-on-kubernetes): fix formatting

### PR DESCRIPTION
**Pull request by michaelbeaumont**
[Before](https://kuma.io/docs/2.0.x/explore/dpp-on-kubernetes/#leaving-the-mesh) vs [after](https://deploy-preview-1173--kuma.netlify.app/docs/2.0.x/explore/dpp-on-kubernetes/#leaving-the-mesh)

### CODE REVIEW

File: app/docs/2.0.x/explore/dpp-on-kubernetes.md [hunk 1]
```diff
@@ -164,42 +164,44 @@ Notice how `kuma.io/service` is built on `<serviceName>_<namespace>_svc_<port>`
...
+   This forces clients to close their connection and reconnect to the new instance.
 
-When Pod is marked as terminated, {{site.mesh_product_name}} CP updates Dataplane object to be unhealthy which will trigger configuration update to all the clients to remove it as a destination.
-This can take a couple of seconds depending on the size of the mesh, available resources for CP, XDS configuration interval, etc.
+You can read the [Kubernetes docs](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination) to learn how Kubernetes handles the `Pod` lifecycle. Here is the summary including the parts relevant for {{site.mesh_product_name}}.
 
-If the application next to the {{site.mesh_product_name}} DP sidecar quits immediately after the SIGTERM signal, there is a high chance that clients will still try to send traffic to this destination.
+Whenever a user or system deletes a `Pod`, Kubernetes does the following:
+1. It marks the `Pod` as terminated.
+1. For every container concurrently it:
```
Comment (lukidzi) [inline app/docs/2.0.x/explore/dpp-on-kubernetes.md:194]: Wrong numbers, it looks like all have 1

File: app/docs/2.0.x/explore/dpp-on-kubernetes.md [hunk 2]
```diff
@@ -164,42 +164,44 @@ Notice how `kuma.io/service` is built on `<serviceName>_<namespace>_svc_<port>`
...
+When the {{site.mesh_product_name}} sidecar receives a SIGTERM signal it:
 
-1) start draining Envoy listeners
-2) wait for the entire draining time
-3) stop the sidecar process
-During the draining process, Envoy can still accept connections however:
-1) It is marked as unhealthy on Envoy Admin `/ready` endpoint
-2) It sends `connection: close` for HTTP/1.1 requests and GOAWAY frame for HTTP/2.
-   This forces clients to close a connection and reconnect to the new instance.
+1. Starts draining Envoy listeners.
```
Comment (lukidzi) [inline app/docs/2.0.x/explore/dpp-on-kubernetes.md:180]: Numbers aren't increasing

Comment (michaelbeaumont) [reply app/docs/2.0.x/explore/dpp-on-kubernetes.md:180]: This is markdown, it makes you not have to ever renumber them if anything changes. It's rendered properly

Comment (michaelbeaumont) [reply app/docs/2.0.x/explore/dpp-on-kubernetes.md:180]: I suppose it might be something we want to agree on if it bothers anyone in the raw markdown

**Review by lukidzi — APPROVED** [review]

**Pull request merged**